app/controllers/use_case_controller.rb

```diff
@@ -66,6 +60,18 @@ def attribute_changes?
     !@model_instance.changed_attributes.blank?
   end
 
+  def build_model_instance
+    if params[:action] == 'new'
+      filtered_params = {}
+    else
+      filtered_params = self.permitted_params
+    end
+    @model_instance = self.model_class.new(filtered_params)
+
+    # TODO unify with how the factories create model instances
```

Comment:
In my defense, these TODOs were added before our decision to put them in stories instead.